eZ/Publish/API/Repository/ContentService.php

```diff
@@ -246,6 +246,17 @@ public function deleteContent(ContentInfo $contentInfo);
      */
     public function createContentDraft(ContentInfo $contentInfo, VersionInfo $versionInfo = null, User $creator = null);
 
+    /**
+     * Counts drafts for a user.
+     *
+     * If no user is given the number of drafts for the authenticated user a returned
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\User $user The user to load drafts from if defined, otherwise drafts for current-user
+     *
+     * @return int The number of drafts ({@link VersionInfo}) owned by the given user
+     */
+    public function countContentDrafts(?User $user = null): int;
```

Comment:
Shouldnt we just stick to `User $user` in cases like those? Look at `loadContentDrafts`

Reply:
I think that we should stay with `?User $user = null` like in other new methods.